lib/omnibus/generator_files/README.md.erb

```diff
@@ -105,17 +105,17 @@ individual build environment using the `kitchen` command.
 
 
 ```shell
-$ bin/kitchen converge ubuntu-1204
+$ bin/kitchen converge ubuntu-1804
 ```
 
 Then login to the instance and build the project as described in the Usage
 section:
 
 ```shell
-$ bundle exec kitchen login ubuntu-1204
+$ bin/kitchen login ubuntu-1804
+[vagrant@ubuntu...] $ .  load-omnibus-toolchain.sh
 [vagrant@ubuntu...] $ cd <%= config[:name] %>
 [vagrant@ubuntu...] $ bundle install
```

Comment:
I think this step is still likely to fail because for some reason we are getting root owned files in `/home/vagrant/.bundle/cache/compact_index/`. My guess is something in the omnibus cookbook is doing this during provisioning, but I haven't dug in further than that.